readme.md

```diff
@@ -212,6 +212,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [cointop](https://github.com/miguelmota/cointop) - The fastest and most interactive terminal based UI application for tracking cryptocurrencies.
 * [colorls](https://github.com/athityakumar/colorls) - Beautify the terminal's `ls` command, with color and font-awesome icons.
 * [ddgr](https://github.com/jarun/ddgr) - DuckDuckGo from the terminal.
+* [df-show](https://github.com/roberthawdon/dfshow) - An interactive directory and file browser with a similar feel to the `ls` command.
```

Comment:
Please drop `An` --> instead, description should be `Interactive directory and file browser with a similar feel to the ls command.`

Reply:
Thanks, I've updated that.